app/assets/stylesheets/landing_page.scss

```diff
@@ -0,0 +1,38 @@
+@import 'bootstrap/dist/css/bootstrap.min';
+
+body {
+  position: fixed;
+  background-color: pink;
+  background-image: url('background1.jpg');
+  height: 100%;
+  width: 100%;
+  color: black;
+}
+
+.container {
```

Comment:
Would prefer not to override Bootstrap class names. Instead, you can use another class name in addition to the BS class name. 

For example, if you renamed this to `.background-wrapper`, you can add this class to to the HTML element without compromising the `.container` class should it be used anywhere else in this layout. 

```html
<div class="container background-wrapper">
  ...
</div>
```

The browser will first render the `.container` style rules, and then apply the `.background-wrapper` style rules. This is considered preventing your styles from polluting global classes (in this case, since `.container` is from Bootstrap, it is considered a global class).